overlay/files-ubuntu-arm-rpi/system/oem/03-setupcon.yaml

```diff
@@ -0,0 +1,7 @@
+name: "Start installer on tty1"
```

Comment:
Maybe we can move this to the generic overlay files and do a check to see if we are on ubuntu? So we dont have to track it separatedly

Reply:
Very good point, I will keep them for this PR so we can merge and opened a discussion on how to address this properly in the future https://github.com/kairos-io/kairos/discussions/1400